journald/journald.cpp

```diff
@@ -176,6 +287,16 @@ int main(int argc, char** argv)
     LOG(WARNING) << warning.message;
   }
 
+  // If the `--user` flag is set, change the UID of this process to that user.
+  if (flags.user.isSome()) {
+    Try<Nothing> result = os::su(flags.user.get());
+
+    if (result.isError()) {
+      EXIT(EXIT_FAILURE)
+        << ErrnoError("Failed to switch user for journald logger").message;
```

Comment:
Is this linux user used for for logrotate, journald, or both? Looks to me like that's who calls the logrotate binary via the `shell` calls. But maybe the journald `subprocess` also uses this?